kustomize/components/google-cloud-operations/otel-collector.yaml

```diff
@@ -0,0 +1,107 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
Is it possible to implement the `securityContext` sections in there like we did here https://github.com/GoogleCloudPlatform/microservices-demo/pull/1046 and here https://github.com/GoogleCloudPlatform/microservices-demo/pull/887?

Reply:
Done.